### ISSUE TITLE
Scene buttons may not line up with scene reset buttons

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/gravity-and-orbits#348]
For https://github.com/phetsims/QA/issues/505. Mostly scene in studio, but discovered in multi wrapper.
When you make one of the scene buttons invisible, the others readjust to take its place. However, the reset buttons for those scenes do not move when this happens, meaning the reset buttons are no longer properly lined up the the scene they control.
![resetsceneindependent](https://user-images.githubusercontent.com/41024075/83924278-7d974580-a741-11ea-9d3a-c8b350134939.gif)

**Comment by samreid** [comment]
The quick fix would be to disable relayout of that section.  That would take 5 minutes and probably be good enough for a dev version.

The better long-term fix would be to reimplement that code around the idea of putting the reset button as part of the radio button components.  That was desired for the original design, but we decided against it in order to speed up turnaround time for a dev version.

Therefore, I recommend to commit the quick fix to unblock a dev version.  But we should try again to reimplement that code for better long term behavior for a later dev version or RC.

**Comment by samreid** [comment]
After the commit, the radio button no longer relayout based on their visibility:

![image](https://user-images.githubusercontent.com/679486/84898860-22c8ec80-b065-11ea-9c38-d26d6c6b5127.png)

Let's call that sufficient for this issue and I'll open a new issue about the long term fix prescribed in the preceding comment.  @KatieWoe can you please confirm on master?  If it works OK, please reassign to @arouinfar to make sure this is acceptable for a dev version.

**Comment by samreid** [comment]
@arouinfar is this acceptable for a dev version?

**Comment by arouinfar** [comment]
> Therefore, I recommend to commit the quick fix to unblock a dev version. But we should try again to reimplement that code for better long term behavior for a later dev version or RC.

@samreid I agree with this 100%. The quick fix is great for the dev version.

**Comment by samreid** [comment]
Thanks, closing.  We will continue in https://github.com/phetsims/gravity-and-orbits/issues/350
